examples/aws/example/models.py

```diff
@@ -1,7 +1,8 @@
-from wagtail.admin.edit_handlers import FieldPanel, StreamFieldPanel
-from wagtail.core import blocks
-from wagtail.core.fields import StreamField
-from wagtail.core.models import Page
+from wagtail import blocks
+from wagtail.admin.panels import FieldPanel
+from wagtail.admin.panels import FieldPanel as StreamFieldPanel
```

Comment:
**Suggestion:** we can drop the StreamFieldPanel import and use FieldPanel in the example. I feel like this aliased import may confuse users reading this example.